### DISCUSSION TITLE
Satisfactory 1.0

### DISCUSSION THREAD

**Discussion by dennybrah** [wolveix/satisfactory-server#261] [General]
Hello @wolveix

I wanted to thank you for this image, pre 1.0 it was working well for my server. 

I saw you have made a few commits today but before I jump the gun, I just wanted to see if it's ready for 1.0 or if I should wait a few till I pull the image?

Thanks!

**Comment by wolveix** [comment]
Hey @dennybrah! It does work with 1.0; however, there's a bug that Coffee Stain are currently working on (I spent the last few hours speaking with a dev directly, and working with members of the community to figure it out). They're hoping to release the patch in a few days, but in the meantime, you can circumvent the issue by setting `net=host` for your container. Please see #260 for more information, thanks!

**Reply by dennybrah** [reply]
Thank you <3

**Comment by bturney** [comment]
@wolveix Also just dropping in to say thank you for all your hard work on this! I don't know how you did it so quickly; I was expecting to wait for a few days. Nice work, and thanks again!

**Reply by wolveix** [reply]
Thanks @bturney, I really appreciate messages like these! It was a stressful day, but I'm pleased everyone is having fun with the game now instead of worrying about technical issues :D
